File: services/identity-service/src/main/java/br/com/deltaglobalbank/identity/infrastructure/persistence/repositories/JpaIssuedTokenAuditRepository.java
package br.com.deltaglobalbank.identity.infrastructure.persistence.repositories;

import java.util.UUID;

import br.com.deltaglobalbank.identity.infrastructure.persistence.entities.IssuedTokenAuditEntity;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface JpaIssuedTokenAuditRepository extends JpaRepository<IssuedTokenAuditEntity, UUID> {
    IssuedTokenAuditEntity findByJti(UUID jti);
}

File: services/identity-service/src/main/kotlin/br/com/deltaglobalbank/identity/infrastructure/persistence/repositories/JpaIssuedTokenAuditRepository.kt
package br.com.deltaglobalbank.identity.infrastructure.persistence.repositories

import br.com.deltaglobalbank.identity.infrastructure.persistence.entities.IssuedTokenAuditEntity
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository
import java.util.UUID

@Repository
interface JpaIssuedTokenAuditRepository : JpaRepository<IssuedTokenAuditEntity, UUID> {
    fun findByJti(jti: UUID): IssuedTokenAuditEntity?
}
